slash-commands/support.js

```diff
@@ -0,0 +1,38 @@
+const { SlashCommand, CommandOptionType, ComponentType, ButtonStyle } = require('slash-create');
+const Discord = require("discord.js");
+const ee = require("../botconfig/embed.json");
+
+module.exports = class SupportCommand extends SlashCommand {
+    constructor(creator) {
+        super(creator, {
+            name: 'support',
+            description: 'Retrieve an invite to the Little Music Discord support server'
+        });
+        this.filePath = __filename;
+    }
+
+    async run(ctx) {
+        await ctx.defer();
+
+        let embed = new Discord.MessageEmbed()
+            .setColor(ee.color)
+            .setTitle(":heart: Join here!")
+            .setFooter(ee.footertext, ee.footericon)
+            .setURL(`https://discord.gg/kUuNQwnvCF`)
+            .setDescription(`[Click here](https://discord.gg/kUuNQwnvCF) (https://discord.gg/kUuNQwnvCF)`)
```

Comment:
Same here, Discord shows what link you are being redirected to as you click on it